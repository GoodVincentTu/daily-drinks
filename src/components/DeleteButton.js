import React, { Component } from 'react';

class DeleteButton extends Component {
  handleDelete = e => {
    e.preventDefault();
    this.props.hadnleDeleteItems(this.props.num);
  }

  render() {
    return (
      <>
        <span onClick={this.handleDelete} >&times;</span>
      </>
    );
  }
}

export default DeleteButton;