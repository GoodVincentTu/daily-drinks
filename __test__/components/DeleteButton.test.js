import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import DeleteButton from '../../src/components/DeleteButton';

describe('<DeleteButton/>', () => {
  it('renders', () => {
    shallow(<DeleteButton/>);
  });

  it('matches the snapshot', () => {
    const wrapper = shallow(<DeleteButton/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
})