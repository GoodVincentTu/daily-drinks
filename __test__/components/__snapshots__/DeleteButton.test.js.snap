// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeleteButton/> matches the snapshot 1`] = `
<Fragment>
  <span
    onClick={[Function]}
  >
    Ã—
  </span>
</Fragment>
`;
